src/client/automation/playback/scroll.js

```diff
@@ -83,20 +83,24 @@ export default class ScrollAutomation {
         var relativePosition = positionUtils.calcRelativePosition(childDimensions, parentDimensions);
 
         if (canShowFullElementWidth) {
-            if (relativePosition.left < 0)
-                fullViewScrollLeft = Math.round(parentDimensions.scroll.left + relativePosition.left);
+            if (relativePosition.left < 0) {
```

Comment:
Don't we check `canShowFullElementWidth` taking into account the indent? @AlexanderMoskovkin